rule_test.go

```diff
@@ -222,6 +222,26 @@ func runRuleListFiltered(t *testing.T, family int, srcNet, dstNet *net.IPNet) {
 				return rs, false
 			},
 		},
+		{
+			name:       "returns one rule filtered by Priority(0) and Table",
+			ruleFilter: &Rule{Priority: 0, Table: 1},
```

Comment:
In my opinion, we should also test the situation where the rule priority is not specified by us when we program the kernel. That happens when we client uses the NewRule() method, and Priority is set to -1.

Reply:
Updated tests to add rule with priority unset, which results into rule creation with priority preceding main table (32766).